src/training/CMakeLists.txt

```diff
@@ -271,6 +271,7 @@ pkg_check_modules(Cairo REQUIRED cairo)
 pkg_check_modules(PangoFt2 REQUIRED pangoft2)
 pkg_check_modules(PangoCairo REQUIRED pangocairo)
 pkg_check_modules(FontConfig REQUIRED fontconfig)
+pkg_check_modules(Glib REQUIRED glib-2.0 gobject-2.0)
```

Comment:
`pango.pc` already includes this requirement, and line 269 requires `pango`, so this should not be needed:

    Requires: glib-2.0 gobject-2.0

Reply:
Actually not always. From my pango.rc:

```
Name: Pango
Description: Internationalized text handling
Version: 1.43.0
Requires.private: glib-2.0 >=  2.38.0, gobject-2.0 >=  2.38.0, fribidi >=  0.19.7, harfbuzz >=  1.4.2, fontconfig >=  2.11.91, freetype2, xrender, xft >=  2.0.0, cairo >=  1.12.10
Libs: -L${libdir} -lpango-1.0
Libs.private: -lm
Cflags: -I${includedir}/pango-1.0
```

What version of Pango and pkgconfig are you using?

Anyway: It is required because there is a **direct** dependency from this target to glib. Relying on a dependency dragging in another dependency is wrong if you require that other dependency directly

Why do you oppose this change so much?